src/Colors/Color.class.st

```diff
@@ -1711,6 +1704,13 @@ Color >> negated [
 		b: 1.0 - self blue
 ]
 
+{ #category : #conversions }
+Color >> nontranslucentColor [
```

Comment:
Maybe `beOpaque` would be a more common name than translucent? But I don't know english enough to know what is the more common between the two.

I checked in an english dictionary online and there was no word `nontranslucent`. Maybe it should be `non translucent` and in that case the method should be called `nonTranslucentColor`?

Reply:
Both good points, @jecisc. Cocoa's UIView has `isOpaque` (getter and setter), so `beOpaque` makes sense here. To fit in with others (i.e., `contrastingColor`, `dominantColor`), how about `opaqueColor`? @Ducasse, you opened the ticket, your thoughts?

Reply:
Both good points, @jecisc. Cocoa's UIView has isOpaque (getter and setter), so beOpaque makes sense here. To fit in with others (i.e., contrastingColor, dominantColor), how about opaqueColor? If not, `nonTranslucentColor` works. @Ducasse, you opened the ticket, your thoughts?

Reply:
Both good points, @jecisc. For example, Cocoa's UIView has `isOpaque` (getter and setter), so `beOpaque` could make sense here. To fit in with other methods in `conversions` (i.e., `contrastingColor`, `dominantColor`), how about `opaqueColor`? If not, `nonTranslucentColor` works. @Ducasse, you opened the ticket, your thoughts?

Reply:
I have no preference just that the previous name was misleading because it is about changing the receiver.

Reply:
I will introduce beOpaque to convey the fact that this is modifying the receiver.
I will do it in another issue to see if the deprecation is not breaking the build